src/sentry/tasks/collect_project_platforms.py

```diff
@@ -1,27 +1,33 @@
 from datetime import timedelta
 
-from django.db.models import Max
 from django.utils import timezone
 
 from sentry.constants import VALID_PLATFORMS
 from sentry.models import Group, Project, ProjectPlatform
 from sentry.tasks.base import instrumented_task
 
 
+def paginate_projects_by_id():
+    id_cursor = -1
+    while True:
+        queryset = Project.objects.filter(id__gt=id_cursor).order_by("id")
```

Comment:
We don't need to build the full `Project` objects. We can add `.values_list("id", flat=True)` here, in which case I'd also change the function name to `paginate_project_ids`. (Seems strange that you can pass it to `project_id__in` either way, but I guess Django is smart enough to figure it out.)